### PR TITLE
First version of logger

### PR DESCRIPTION
**Pull request by code4trade**
The PR adds a simple/first version to use logger in our microservices.

### CODE REVIEW

**Review by rafaelescrich — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
\ No newline at end of file
+<img src="https://raw.githubusercontent.com/Yoonit-Labs/android-yoonit-camera/development/logo_cyberlabs.png" width="300">
+
+# logger
+
+Logger is a Logrus extension to quickly use as a Singleton when it is an advantage to your project.
+
+## How it works
+
+The logger initializes independently with the function init(), than you can use their functions in any package.
```
Comment (cleberzavadniak) [inline README.md:9]: "**then** you can use **its** functions"

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
-# logger
\ No newline at end of file
+<img src="https://raw.githubusercontent.com/Yoonit-Labs/android-yoonit-camera/development/logo_cyberlabs.png" width="300">
+
+# logger
+
+Logger is a Logrus extension to quickly use as a Singleton when it is an advantage to your project.
```
Comment (cleberzavadniak) [inline README.md]: Is it and extension or a wrapper?

Besides, I would get rid of this "when it is an advantage to your project" part.

I would write, actually: "*Logger is an opinionated wrapper around Logrus that is extremely simple to use.*"

Comment (cleberzavadniak) [reply README.md]: **And** "It also has automatic integration with Sentry".

File: logger.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	dsn, exists := os.LookupEnv("SENTRY_DSN")
+
+	if exists {
+		hook, err := logrus_sentry.NewSentryHook(dsn, []logrus.Level{
+			logrus.PanicLevel,
+			logrus.FatalLevel,
+			logrus.ErrorLevel,
+		})
+		if err != nil {
+			logger.Warnf("It was not possible to enable Sentry handler: %w", err)
```
Comment (cleberzavadniak) [inline logger.go]: Is it simply a warning? Is the function supposed to continue normally after this?

File: logger.go [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+	"os"
+
+	"github.com/evalphobia/logrus_sentry"
+	"github.com/sirupsen/logrus"
+)
+
+var logger *logrus.Logger
+
+// F is a map used to log custom fields.
+type F map[string]interface{}
```
Comment (cleberzavadniak) [inline logger.go]: Where is it being used?

Comment (code4trade) [reply logger.go]: It's used in example below, this example is on readme.

```go
package main

import log "github.com/cyberlabsai/logger"

func main() {
    // Note that the F is just a easy access to a map.
    log.WithFields(log.F{
		"user_id": 1,
		"func":    "customMessage",
		"message": "my custom message",
	}).Info("When i log with custom fields")
}
```

**Pull request merged**